docs/cheqd-cli/cheqd-cli-identity-tutorial.md

```diff
@@ -0,0 +1,413 @@
+z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe3Nmod35uua9TE
+```
+
+  
+
+And the response will be:
+
+  
+
+```
+z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe3Nmod35uua9TE
+```
+
+  
+
+### 3\. Create unique-id for our DID
+
+  
+
+To create a `unique-id` for our DID we can use first 32 symbols of `multibase58` representation of our public key as \`unique-id\`.
+
+For example, we can truncate previous one:
+
+```
+$ printf '%.32s\n' `cheqd-noded debug encoding base64-multibase58 <pub_key_base_64>`
+```
+
+  
+
+The result for our example will be `z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe` , so let's use it as our `unique-id` in our DIDDoc.
+
+  
+
+### 4\. Compile DIDDoc
+
+  
+
+Next we can compile our DIDDoc.
+
+  
+
+Copy and paste the template below into your terminal. We will add additional required information into the blank fields `<xxxxx>` in the next steps.
+
+  
+
+```
+{
+  "id": "did:cheqd:<namespace>:<unique-id>",
+  "verification_method": [
+    {
+      "id": "did:cheqd:<namespace>:<unique-id>#<key-alias>",
+      "type": "Ed25519VerificationKey2020",
+      "controller": "did:cheqd:<namespace>:<unique-id>",
+      "public_key_multibase": "<verification-public-key-multibase>"
+    }
+  ],
+  "authentication": [
+    "did:cheqd:<namespace>:<unique-id>#<auth-key-alias>"
+  ]
+}
+```
+
+Within this template we will be required to enter a number of fields
+
+  
+
+Where:
+
+*   `<namespace>` - for now it can `testnet` or `mainnet` . For this flow we use `testnet`
+*   `<unique-id>` - identifier, created on the [step](#3-create-unique-id-for-our-did)
+*   `<key-alias>` - a key alias for the verification method identifier
+*   `<verification-public-key-multibase>` - result of this [step](#2-get-multibase58-string)
+*   `<auth-key-alias>` - alias of authentication key.
+
+In our example:
+
+*   `did:cheqd:<namespace>:<unique-id>` - would be `did:cheqd:testnet:z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe`
+*   `did:cheqd:<namespace>:<unique-id>#<key-alias>` - `did:cheqd:testnet:z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe#key1`
+*   `<verification-public-key-multibase>` - key from this [step](#2-get-multibase58-string). As result `z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe3Nmod35uua9TE`
+*   `did:cheqd:<namespace>:<unique-id>#<auth-key-alias>` - let it be:
+
+`did:cheqd:testnet:z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe#key1`
+
+  
+
+#### After these preparations, the base DIDDoc will look like:
+
+  
+
+```
+{
+  "id": "did:cheqd:testnet:z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe",
+  "verification_method": [
+    {
+      "id": "did:cheqd:testnet:z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe#key1",
+      "type": "Ed25519VerificationKey2020",
+      "controller": "did:cheqd:testnet:z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe",
+      "public_key_multibase": "z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe3Nmod35uua9TE"
+    }
+  ],
+  "authentication": [
+    "did:cheqd:testnet:z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe#key1"
+  ]
+}
+```
+
+  
+
+We recommend you store this DIDDoc in a separate file, like `json.txt` and inject it while running the command for sending.
```

Comment:
Change to diddoc.json or similar (will activate highlighting/formatting if supported by the text editor being used)